README.md

```diff
@@ -61,26 +59,6 @@ dev/test
 
 These tests provide a full suite of unit and integration tests for the `xmtpd` repo to help ensure and maintain correctness of the code over time and to avoid regressions as the code evolves. You can explore the tests by taking a look at any files with the suffix `_test.go`.
 
-## Monitor the node
-
-The `xmtpd` node build provides two options for monitoring your node.
-
-- To access your local Prometheus instance to explore node metrics, run:
-
-  ```sh
-  open http://localhost:9090
```

Comment:
FWIW localhost:9090 still does exist in the current config. I'm having problems actually getting metrics into the prometheus dashboard on my mac. But the dashboard is live.

Maybe we want to just shorten this section, and mention that the node needs to be run with the `--metrics.enable` flag

Reply:
will do